app/src/main/java/mod/hey/studios/activity/managers/java/ManageJavaActivity.java

```diff
@@ -49,48 +50,13 @@ public class ManageJavaActivity extends BaseAppCompatActivity {
     // works for both Java & Kotlin files
     private static final String PACKAGE_DECL_REGEX = "package (.*?);?\\n";
 
-    private static final String ACTIVITY_TEMPLATE =
-            "package %s;\n" +
-                    "\n" +
-                    "import android.app.Activity;\n" +
-                    "import android.os.Bundle;\n" +
-                    "\n" +
-                    "public class %s extends Activity {\n" +
-                    "\n" +
-                    "    @Override\n" +
-                    "    protected void onCreate(Bundle savedInstanceState) {\n" +
-                    "        super.onCreate(savedInstanceState);\n" +
-                    "    }" +
-                    "\n" +
-                    "}\n";
-
-    private static final String CLASS_TEMPLATE =
-            "package %s;\n" +
-                    "\n" +
-                    "public class %s {\n" +
-                    "    \n" +
-                    "}\n";
-
-    private static final String KT_ACTIVITY_TEMPLATE =
-            "package %s\n" +
-                    "\n" +
-                    "import android.app.Activity\n" +
-                    "import android.os.Bundle\n" +
-                    "\n" +
-                    "class %s : Activity() {\n" +
-                    "\n" +
-                    "    override fun onCreate(savedInstanceState: Bundle?) {\n" +
-                    "        super.onCreate(savedInstanceState)\n" +
-                    "    }" +
-                    "\n" +
-                    "}\n";
-
-    private static final String KT_CLASS_TEMPLATE =
-            "package %s\n" +
-                    "\n" +
-                    "class %s {\n" +
-                    "    \n" +
-                    "}\n";
+    private static final String ACTIVITY_TEMPLATE = "package %s;\n" + "\n" + "import android.app.Activity;\n" + "import android.os.Bundle;\n" + "\n" + "public class %s extends Activity {\n" + "\n" + "    @Override\n" + "    protected void onCreate(Bundle savedInstanceState) {\n" + "        super.onCreate(savedInstanceState);\n" + "    }" + "\n" + "}\n";
+
+    private static final String CLASS_TEMPLATE = "package %s;\n" + "\n" + "public class %s {\n" + "    \n" + "}\n";
+
+    private static final String KT_ACTIVITY_TEMPLATE = "package %s\n" + "\n" + "import android.app.Activity\n" + "import android.os.Bundle\n" + "\n" + "class %s : Activity() {\n" + "\n" + "    override fun onCreate(savedInstanceState: Bundle?) {\n" + "        super.onCreate(savedInstanceState)\n" + "    }" + "\n" + "}\n";
+
+    private static final String KT_CLASS_TEMPLATE = "package %s\n" + "\n" + "class %s {\n" + "    \n" + "}\n";
 
```

Comment:
Why did you do that?

Reply:
automatically fomatted by android studio